src/optiondefs.h

```diff
@@ -2769,7 +2769,7 @@ static struct vimoption options[] =
 			    {(char_u *)"",
 				 (char_u *)"'100,<50,s10,h,rdf0:,rdf1:,rdf2:"}
 #else
-			    {(char_u *)"", (char_u *)"'100,<50,s10,h"}
+			    {(char_u *)"", (char_u *)"'100,<500,s100,h"}
```

Comment:
If we change the default, I think we should also increase it for at least Windows (see above).

Reply:
fixed that

Reply:
options.txt should be also updated accordingly:
https://github.com/vim/vim/blob/701ad50a9efcf0adfe6d787b606c4e4dbd31f26d/runtime/doc/options.txt#L9032-L9034